.github/workflows/test-commit.yml

```diff
@@ -0,0 +1,24 @@
+name: Test
+on: [pull_request]
+jobs:
+  test:
+    name: Run tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install
+        run: cd node && npm ci
+      - name: Run audit test
+        run: cd node && npm run test:audit
+      - name: Run format test
+        run: cd node && npm run test:format
+      - name: Run lint test
+        run: cd node && npm run test:lint
+      - name: Run types test
+        run: cd node && npm run test:types
+      - name: Run license test
+        run: cd node && npm run test:license
+      - name: Run tests
+        run: cd node && npm run test
+
+  
```

Comment:
tiny thing but let's trim this down to just one newline at the end of the file (maybe we can get prettier to format/enforce this for yaml?)